api/src/pcapi/routes/backoffice/offers/blueprint.py

```diff
@@ -734,12 +733,10 @@ def _batch_reject_offers(offer_ids: list[int]) -> None:
             cancelled_bookings = bookings_api.cancel_bookings_from_rejected_offer(offer)
 
             if cancelled_bookings:
-                # FIXME: La notification indique que l'offreur a annulé alors que c'est la fraude
-                # TODO(PC-23550): SPIKE avec marketing https://passculture.atlassian.net/browse/PC-23550
-                # Il faudrait utiliser send_booking_cancellation_emails_to_user_and_offerer et retirer cette notification soit la déplacer dedans, mais un mail est mieux (TBD)
-                push_notification_job.send_cancel_booking_notification.delay(
-                    [booking.id for booking in cancelled_bookings]
-                )
+                for booking in cancelled_bookings:
```

Comment:
Tu peux retirer la condition `if cancelled_bookings:` au-dessus car `cancel_bookings_from_rejected_offer` ne renvoie jamais `None` et la liste vide sera bien gérée dans le `for`.